source/_includes/pup40_platforms_redhat_like.markdown

```diff
@@ -1,10 +1,10 @@
 We publish official packages and run automated testing on the following versions for Puppet 4.x:
 
-* Enterprise Linux 6 (also supported by [Puppet Enterprise][peinstall])
 * Enterprise Linux 7 (also supported by [Puppet Enterprise][peinstall])
-
+* Enterprise Linux 6 (also supported by [Puppet Enterprise][peinstall])
+* Enterprise Linux 5 (also supported by [Puppet Enterprise][peinstall])
```

Comment:
@underscorgan let's come up with a procedure for this once I'm back in the states